geth/account/interfaces.go

```diff
@@ -0,0 +1,45 @@
+package account
+
+import (
+	"crypto/ecdsa"
+	"github.com/ethereum/go-ethereum/accounts"
+	"github.com/ethereum/go-ethereum/accounts/keystore"
+	gethcommon "github.com/ethereum/go-ethereum/common"
+	"github.com/status-im/status-go/extkeys"
+)
+
+type accountNode interface {
+	AccountKeyStore() (accountKeyStorer, error)
+	AccountManager() (gethAccountManager, error)
+	WhisperService() (whisperService, error)
+}
+
+type gethAccountManager interface {
+	Wallets() []accounts.Wallet
+}
+
+type accountKeyStorer interface {
+	AccountDecryptedKey(account accounts.Account, password string) (accounts.Account, *keystore.Key, error)
+	IncSubAccountIndex(account accounts.Account, password string) error
+	ImportExtendedKey(extKey *extkeys.ExtendedKey, password string) (accounts.Account, error)
+	Accounts() []accounts.Account
+}
+
+type whisperService interface {
+	DeleteKeyPairs() error
+	SelectKeyPair(key *ecdsa.PrivateKey) error
+}
+
+type subAccountFinder interface {
+	Find(keyStore accountKeyStorer, extKey *extkeys.ExtendedKey, subAccountIndex uint32) ([]accounts.Account, error)
+}
+
+type keyFileFinder interface {
+	Find(keyStoreDir string, addressObj gethcommon.Address) ([]byte, error)
+}
+
+// importExtendedKey processes incoming extended key, extracts required info and creates corresponding account key.
```

Comment:
Linter won't accept this comment.

Reply:
@themue Could you share your linter log? I don't see any problems with it(https://travis-ci.org/status-im/status-go/jobs/296314709)

Reply:
My _inside_ linter, sorry. 😁 I'm so used to even comment private types and function in a godoc matching way. But now I tested my editor linter and he allows it this way in private identifiers, yes. Sorry. 

Still would prefer a naming more matching to the interface below.